### ISSUE TITLE
standardize dataset naming conventions

### ISSUE THREAD

**Issue by kirk86** [pytorch/vision#1305]
Please standardize at least the naming convention for all datasets when using DataLoader so that data are in `loader.dataset.data` and labels in `loader.dataset.targets`.
MWE:
```
loader_cifar = torch.utils.DataLoader("CIFAR10")
loader_cifar.dataset.data <-- contains the data
loader_cifar.dataset.targets <-- contains targets

loader_svhn = torch.utils.DataLoader("SVHN")
loader_svhn.dataset.data <-- contains the data
loader_svhn.dataset.labels <-- contains targets
```
This can be really confusing and error prone when developing code to run on multiple datasets.

**Comment by pmeier** [comment]
See #1080.

**Comment by kirk86** [comment]
Thanks, where are we with that? Any progress? 
I've already added my two cents on that issue as well.
